modelexpress_client/python/modelexpress/engines/vllm/artifacts.py

```
@@ -336,30 +336,28 @@ def _set_extra_if_present(
 def _vllm_artifact_ready_fn(
     source_roots: tuple[ArtifactCacheRoot, ...],
     health_ready_fn=None,
+    ctx: LoadContext | None = None,
 ):
     if health_ready_fn is None:
         # Resolve the engine hook at call time so existing tests and diagnostics
         # can replace _vllm_health_ready after constructing the readiness check.
-        health_ready_fn = lambda: _vllm_health_ready()
+        health_ready_fn = lambda: _vllm_health_ready(ctx)
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `health_ready_fn` as a `def`

(E731)